Minimal_test/Test.html

```diff
@@ -0,0 +1,16 @@
+<html>
+  <head>
+	  <title>Santiagos Webseite</title>
+  </head>
+  <body>
+    <h1 id="HelloWorld">hello world</h1>
+  <script>
+    if (document.getElementById("HelloWorld").innerHTML == "hello world") {
+      alert("true")
+    }
+    else {
+      alert("false")
+    }
+  </script>
```

Comment:
sorry: ich hatte mich im Ticket nicht gut ausgedrückt: die Aufgabe sollte sein, das mit mocha.js zu machen, so wie in dem verlinkten Artikel. Hintergrund ist, dass ich für die späteren Tests auch diese Test-Framework nutzen möchte